src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    lastKeyAp: 'Nothing was pressed yet',
```

Comment:
Consider renaming 'lastKeyAp' to 'lastKeyPressed' for better readability and to match the task description.